src/lib/utils/cpuid/cpuid_arm.cpp

```diff
@@ -16,6 +16,10 @@
 #else
   #include <botan/internal/os_utils.h>
 
+#if defined(BOTAN_TARGET_OS_HAS_GETAUXVAL) || defined(BOTAN_TARGET_OS_HAS_ELF_AUX_INFO)
+  #include <sys/auxv.h>
```

Comment:
Why add this include? It should only be needed if we are invoking `getauxval` directly instead of through `OS::` wrapper, right?

Reply:
Not so much about getauxval but more AT_DBCACHESIZE

Reply:
Ah got it thanks